src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Compiler/AddConstraintValidatorsPass.php

```diff
@@ -24,6 +24,7 @@ public function process(ContainerBuilder $container)
 
         $validators = array();
         foreach ($container->findTaggedServiceIds('validator.constraint_validator') as $id => $attributes) {
+            $validators[$id] = $id;
```

Comment:
using the id does not make sense. It should use the service class instead (which is what is returned by ``getValidatedBy`` by default